src/test/scala/org/constellation/p2p/PeerAPITest.scala

```diff
@@ -71,19 +71,17 @@ class PeerAPITest
       val req = FinishedCheckpoint(CheckpointCache(None), Set.empty)
 
       Post("/finished/checkpoint", req) ~> addHeader("ReplyTo", reply) ~> peerAPI.postEndpoints ~> check {
-        status shouldEqual StatusCodes.OK
-        responseAs[FinishedCheckpointAck] shouldEqual FinishedCheckpointAck(true)
+        status shouldEqual StatusCodes.Accepted
       }
 
       //      requires mockito 1.4.x and migrating all IdiomaticMockitos
       //      peerAPI.makeCallback(*, *) wasCalled (once within 2.seconds)
     }
 
-    "return acknowledge message on finishing checkpoint and make no reply with callback" in {
+    "return accepted on finishing checkpoint and make no reply with callback" in {
```

Comment:
why there are two tests and one says "reply with callback" and second "make no reply with callback"
IMO one of the tests is missing a proper case message - which behavior is default and which is because of sth

Reply:
I've changed  the other test title